dotnet/src/IntegrationTests/Connectors/OpenAI/OpenAICompletionTests.cs

```diff
@@ -184,7 +184,7 @@ public async Task OpenAIHttpRetryPolicyTestAsync(string prompt, string expectedO
 
         this._kernelBuilder.Services.AddSingleton<ILoggerFactory>(this._testOutputHelper);
         this._kernelBuilder
-            .AddOpenAITextGeneration(
+            .AddOpenAIChatCompletion(
                 serviceId: openAIConfiguration.ServiceId,
                 modelId: openAIConfiguration.ModelId,
```

Comment:
Should not the `ChatModelId`be used instead of `ModelId` ?

Reply:
Thanks, resolved.